index.js

```diff
@@ -265,5 +265,5 @@ function euclideanMod (numerator, denominator) {
   return result < 0 ? result + denominator : result
 }
 
-module.exports = v
-v.Vec3 = Vec3
+export { Vec3, v }
+export default v
```

Comment:
IIUC you want to maintain `require('vec3')` returns a function. Then it'd need

```suggestion
v.Vec3 = Vec3
export default v  // So that import v from 'vec3' works.
export { v as 'module.exports' }  // So that const v = require('vec3') works.
export { Vec3 }  // So that import { Vec3 } from 'vec3' and const { Vec3 } = require('vec3') work.
```